src/app/edit-dialog/edit-dialog.component.ts

```diff
@@ -0,0 +1,15 @@
+import { Component, OnInit } from '@angular/core';
```

Comment:
keep same code-style allover the project (in this case ' vs ")

Reply:
done